### ISSUE TITLE
Speak window occasionally disappears

### ISSUE THREAD

**Issue by benkuhn** [speak/issues#152]
Sometimes people call me and the Speak window doesn't appear (but we can still hear each other). I've tried various things to find the window (e.g. click on the dock and command+backtick) but have not had success. I'm not sure if it's not there or just hidden behind other windows.

This isn't too inconvenient for audio calls, but it's kind of annoying for video. My workaround for now is to call them from my end.

I think I've noticed this happening more often when I'm having network difficulties, but I'm not totally sure about that. Sorry for the lack of details--let me know if there's anything I can check the next time it happens!

**Comment by tommoor** [comment]
Thanks Ben, appreciate the report - we have some changes coming to the call widget that might mitigate this although we haven't specifically found this issue ourselves so far.

If you come across any further details then let me know :smile:

**Comment by tommoor** [comment]
Ben, I accidentally reproduced this earlier today in my local development environment - in my case it was due to a delayed connected message from the server, so that lines up with your network difficulties theory.

**Comment by tommoor** [comment]
This should be all fixed up in 0.6.0 today

**Comment by benkuhn** [comment]
Hasn't happened to me since then :D

**Comment by tommoor** [comment]
:sweat_smile: phew, thanks for following up!

**Comment by benkuhn** [comment]
I just saw it again :( Not sure if it's the same bug or not, but commenting here.

I called someone on my team yesterday and the window showed up while it was attempting to connect, but then disappeared once I was connected.

Clicking on his image in the dock again caused the window to come up again with "connecting..."

When he tried to loop a third person in, there was an audio connection between all three of us, but the third person couldn't see that I was in the call. Spooky.
